ldi-core/ngsiv2-to-ld/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/valuobjects/valueproperties/PostalAddressValue.java

```diff
@@ -0,0 +1,25 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.valuobjects.valueproperties;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static be.vlaanderen.informatievlaanderen.ldes.ldi.config.NgsiV2ToLdMapping.*;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class PostalAddressValue extends ObjectValue {
```

Comment:
this seems very niche for a general converter.. This is in the original code or in the algorithm?